tools/package.json

```
@@ -10,12 +10,17 @@
     "templates"
   ],
   "scripts": {
-    "build": "tsc",
-    "watch": "tsc --watch",
+    "build": "taskr release",
+    "watch": "taskr",
     "clean": "rm -rf build cache",
     "et": "node bin/expotools.js",
     "lint": "eslint ./src"
```

```suggestion
    "lint": "eslint ./src taskfile.js taskfile-swc.js"
```

Probably we should add the new task files to lint.